README.md

```diff
@@ -0,0 +1,154 @@
+# fluid-lint-all
+
+This package runs the standard linting checks used within the
+[Fluid community](https://fluidproject.org).
+
+## Requirements
+
+If you want to use this package without installing it, you'll need to install [`npx`](https://www.npmjs.com/package/npx).
+
+## Usage
+
+### Adding the Checks as a Dependency of Your Project
+
+1. Install the package using a command like `npm install --save-dev fluid-lint-all`.
+2. Add a script to your package.json, something like: ```"lint": "fluid-lint-all"```
+3. Run the checks using a command like `npm lint`.
+
+### Running the Checks Using `npx`
+
+To use this package without installing it as a dependency, use a command like:
+
+```npx fluid-lint-all```
+
+Although it's not required, it is recommended that you install the
+[standard ESLint configuration used within the community](https://github.com/fluid-project/eslint-config-fluid).  If you
+choose not to, you must at least have some kind of `.eslintrc.json` in your repository.  Otherwise, some of the ESLint
+checks will fail for lack of a configuration file.
+
+## Tests
+
+To run the tests in this package, use a command like `npm test`.
+
+## Checks Available
+
+| Task                       | Description |
+| -------------------------- | ----------- |
+| eslint                     | Use [ESLint](https://eslint.org) to lint both javascript and markdown content. |
+| eslint.js                  | Check the validity and formatting of Javascript files. |
```

Comment:
The `eslint.json` check is not listed here.

Reply:
Good catch.  This was previously a separate task, I removed the description when I collapsed it under `eslint`.  I put back in similar wording.